### PR TITLE
Changing "DisplayAdmin" role name to "Administrator" [stage-2]

### PR DESCRIPTION
**Pull request by stulees**

### CODE REVIEW

**Comment by stulees** [comment]
@Rise-Vision/delivery Please review, thanks

**Comment by settinghead** [comment]
:+1:

**Comment by settinghead** [comment]
@stulees don't merge yet, there is more.

**Comment by settinghead** [comment]
@stulees I remember at one time we actually switched from "Administrator" to "DisplayAdmin" because it's a less ambiguous name. That said I think the correct approach here is to leave "DisplayAdmin" as an internal representation, but change all appearances in the UI to "Administrator" (as far as I can recall the only relevant place is labels in drop-down select boxes in "Add User" and "edit user" dialogs.

@tejohnso FYI

**Comment by stulees** [comment]
@settinghead Makes sense and also means we can leave values in database as they are. I'll make the change, thanks.

**Comment by stulees** [comment]
@settinghead Please review again, thanks

File: src/components/display_users/add_user.html [hunk 1]
```diff
@@ -25,7 +25,7 @@ <h4 ng-hide="invitationSent" class="modal-title" id="modal-title">Add User to Di
               required
               name="role"
               class="form-control">
-              <option>DisplayAdmin</option>
+              <option>Administrator</option>
```
Comment (settinghead) [inline src/components/display_users/add_user.html]: @stulees I think this value (`Administrator`) would be directly committed to database, which would be inconsistent with `DisplayAdmin` that rest of the system uses. Instead, I think you can do something like `<option value="DisplayAdmin">Administrator</option>` which separates label value from form value.

**Comment by stulees** [comment]
@settinghead Good catch, change made, please review.

**Comment by settinghead** [comment]
@stulees all good now. Thanks. :+1:

**Pull request merged**